packages/core/src/common/logger.ts

```diff
@@ -9,22 +9,22 @@ import { inject, injectable, optional } from 'inversify';
 import { LoggerWatcher } from './logger-watcher';
 import { ILoggerServer } from './logger-protocol';
 
-export enum LogLevel {
-    FATAL = 60,
-    ERROR = 50,
-    WARN = 40,
-    INFO = 30,
-    DEBUG = 20,
-    TRACE = 10
+export namespace LogLevel {
```

Comment:
why not the enum?

Reply:
I just reverted the commit but i can put back the enum